x-pack/plugins/infra/public/hooks/use_http_request.tsx

```diff
@@ -19,11 +19,11 @@ export function useHTTPRequest<Response>(
   body?: string | null,
   decode: (response: any) => Response = (response) => response,
   fetch?: HttpHandler,
-  toastWarning?: (input: ToastInput) => void
+  toastDanger?: (input: ToastInput) => void
 ) {
   const kibana = useKibana();
   const fetchService = fetch ? fetch : kibana.services.http?.fetch;
-  const toast = toastWarning ? toastWarning : kibana.notifications.toasts.warning;
+  const toast = toastDanger ? toastDanger : kibana.notifications.toasts.danger;
```

Comment:
What's the significance of this warning / danger change?

Reply:
I forgot to update this with something more general.  I changed it to danger because that's going to be the default now instead of warning.  But I'm thinking to name it something like "toastInput"